src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/Histogram.java

```diff
@@ -0,0 +1,130 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator;
+
+
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import java.util.Arrays;
+
+/**
+ * Created by gauthier on 11/1/16.
+ *
+ * Ported by emeryj 7/24/17
+ */
+public class Histogram {
```

Comment:
Whoops looks like I missed that, for now I will say yes, because there still would need to be an adapter for reading/writing the thing which would require a bunch of codewacking